library/core/src/slice/mod.rs

```diff
@@ -937,11 +933,11 @@ impl<T> [T] {
     #[unstable(feature = "slice_as_chunks", issue = "74985")]
     #[inline]
     pub unsafe fn as_chunks_unchecked<const N: usize>(&self) -> &[[T; N]] {
-        debug_assert_ne!(N, 0);
-        debug_assert_eq!(self.len() % N, 0);
-        let new_len =
-            // SAFETY: Our precondition is exactly what's needed to call this
-            unsafe { crate::intrinsics::exact_div(self.len(), N) };
+        // SAFETY: Our precondition is exactly what's needed to call this
```

Comment:
these SAFETY comments are a bit weak

Reply:
This one is a bit odd as safety comments go. Do you have a suggestion?

Reply:
```suggestion
        // SAFETY: caller must guarantee that `N` exactly divides the slice length
```